src/components/Trade/Components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -74,7 +77,14 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = ({ isOpen, isLoading }) =
 
         const totalReceiveAmountCryptoPrecision = bnOrZero(buyAmountBeforeFeesCryptoPrecision)
           .minus(totalProtocolFeeCryptoPrecision)
-          .times(bn(1).minus(bnOrZero(quote.recommendedSlippage ?? DEFAULT_SLIPPAGE)))
+          .times(
+            bn(1).minus(
+              bnOrZero(
+                quote.recommendedSlippage ??
+                  getDefaultSlippagePercentageForSwapper(swapperWithMetadata.swapper.name),
```

Comment:
I wonder if it might be nicer to make the swappers define their default percentage rather than shifting this concern outside of them in this helper. e.g add a getter on the swapper interface so we can do `swapperWithMetadata.swapper.defaultSlippage`